test/common/index.js

```diff
@@ -804,3 +804,9 @@ exports.runWithInvalidFD = function(func) {
 
   exports.printSkipMessage('Could not generate an invalid fd');
 };
+
+Object.defineProperty(exports, 'isFSSensitiveCI', {
+  get: function() {
+    return process.env['TRAVIS'] === 'true';
```

Comment:
How about just directly using `process.env.TRAVIS` ?

Reply:
I like the verbosity. But I agree `return process.env.TRAVIS` is also nice... Thinking...